src/middlewared/middlewared/migration/0003_iscsi_vendor.py

```diff
@@ -1,12 +1,11 @@
 async def migrate(middleware):
-    is_freenas = await middleware.call('system.is_freenas')
     extents = await middleware.call('iscsi.extent.query', [['vendor', '=', None]])
     for extent in extents:
         await middleware.call(
             'datastore.update',
             'services.iscsitargetextent',
             extent['id'], {
-                'iscsi_target_extent_vendor': 'FreeNAS' if is_freenas else 'TrueNAS'
+                'iscsi_target_extent_vendor': 'TrueNAS'
```

Comment:
This may need more thought, we want older systems to keep using FreeNAS for upgrades